frontend/.env.development

```diff
@@ -1,3 +0,0 @@
-VUE_APP_FRONTEND=http://127.0.0.1:8080
-VUE_APP_FUNCTION_KEY=SteUyT0o2YSt4JW8k2qZ5ShTBcQYQj1pxE8cO7oTrYI4ps92/53o9w==
-VUE_APP_BACKEND=https://localhost:5001

```

Comment:
Yea, I got mixed feelings about keeping vs removing this. This would be needed when they do local development right (i.e. API endpoint)? How about removing the endpoints but keep the variables with a placeholder for them to fill? 

We should probably take out the function key either way but keep the endpoints variables?

I dont want to add more change since this is kind of out of scope for the auth part, if you like I can clean this up in a separate PR  along with the VUE_APP_FUNCTION_KEY since its not needed in our arch.

Reply:
Agreed - removing file for now.